### DISCUSSION TITLE
若单句歌词因过长而换行，则逐词歌词会有显示问题

### DISCUSSION THREAD

**Discussion by KeeLee2005** [amll-dev/applemusic-like-lyrics#220] [General]
首先请允许我表达对开发者无私奉献的感谢。

我遇到的问题是：
如果单句歌词因为过长而换行，则可能会两行歌词同步播放逐词（如图一二），如果第二行歌词较短则不会出现这个问题（如图三）。问题在重启网易云后仍稳定复现。
![image](https://github.com/Steve-xmh/applemusic-like-lyrics/assets/92454987/56000d50-ac2c-44e8-ad54-bcf908e83617)
![image](https://github.com/Steve-xmh/applemusic-like-lyrics/assets/92454987/f9d027c4-b64b-4fe1-9ed7-9b273b0304a4)
![image](https://github.com/Steve-xmh/applemusic-like-lyrics/assets/92454987/994e9144-278c-4866-9a30-4a1bfc6e4143)

网易云音乐版本: 2.10.12 (Build:201849)
BetterNCM版本: 1.3.2
Apple Music-like Lyrics版本: 3.0.3 (main-f21ae39f)
系统版本: Windows 10 22H2 Build19045.3803

祝 万事顺意。

**Comment by Steve-xmh** [comment]
已确认是问题，开发分支里应该修正了
